server/build/webpack.js

```diff
@@ -178,6 +189,7 @@ export default async function createCompiler (dir, { hotReload = false, dev = fa
       ],
       loaders
     },
+    devtool: dev ? 'inline-source-map' : false,
```

Comment:
I tested with `cheap-module-eval-source-map` and for me this is the fastest setting with correct breakpoints (Chrome on OS X)